### DISCUSSION TITLE
有必要明确一下标题或介绍

### DISCUSSION THREAD

**Discussion by xlab6** [vue-mini/vue-mini#75] [Q&A] [answered]
**至少明确一下标题或介绍**

看了一圈网站、文档，没发现任何针对性的小程序平台描述。现状是小程序 !== 微信小程序

**Comment by yangmingshan** [comment, marked as answer]
我建了个 issue #76，我会在文档加一篇写清楚。

**Comment by yangmingshan** [comment]
https://vuemini.org/guide/#%E6%94%AF%E6%8C%81%E7%9A%84%E5%B9%B3%E5%8F%B0
